rules/evasion/covert-location/php_image.yara

```diff
@@ -7,8 +7,13 @@ rule php_image_include: critical {
     $php     = "<?php"
     $include = /include\s*\(\s*[^\.]+\.(png|jpg|gif|bmp)/
 
+    // https://github.com/symfony/symfony/blob/7.4/src/Symfony/Component/ErrorHandler/Resources/views/exception_full.html.php#L9
+    // https://github.com/symfony/symfony/blob/7.4/src/Symfony/Component/ErrorHandler/Resources/assets/images/favicon.png.base64
+    $not_symfony1 = "include('assets/images/favicon.png.base64')"
+    $not_symfony2 = "<a href=\"https://symfony.com/doc/<?= Symfony\\Component\\HttpKernel\\Kernel::VERSION; ?>/index.html\">"
+
   condition:
-    filesize < 5242880 and all of them
+    filesize < 5242880 and all of them and none of ($not*)
```

Comment:
I've felt like I should ask this about other false positive mitigations we've been putting in place, but should the logic here be instead:

`and not all of $(not*)` as in, ignore if all the exception rules are matched rather than any one of them?

(I can open this as a separate issue if we'd like to discuss this as a general project principle.)

Reply:
I've been thinking about that. We can workshop this 🙂